tests/migration/migration.go

```diff
@@ -1697,7 +1697,6 @@ var _ = Describe(SIG("VM Live Migration", decorators.RequiresTwoSchedulableNodes
 						"Failed":         BeTrue(),
 						"StartTimestamp": Not(BeNil()),
 						"EndTimestamp":   Not(BeNil()),
-						"Completed":      BeTrue(),
```

Comment:
Imho this is wrong. It seems there was some offline discussion that didn't bubble up. @fossedihelm